src/main/java/uk/gov/hmcts/reform/sscs/model/hearings/HearingDetails.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.sscs.model.hearings;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@SuppressWarnings("PMD.TooManyFields")
+public class HearingDetails {
+
+    private boolean autolistFlag;
+
+    private String hearingType;
+
+    private HearingWindow hearingWindow;
+
+    private Integer duration;
```

Comment:
Why Integer over int?

Reply:
Primitives can't be null, if there's a chance the value will ever be null it's safer